src/wallet/api/wallet.cpp

```diff
@@ -605,18 +603,18 @@ bool WalletImpl::setPassword(const std::string &password)
     return m_status == Status_Ok;
 }
 
-std::string WalletImpl::address() const
+std::string WalletImpl::address(uint32_t account_index, uint32_t address_index) const
```

Comment:
Oh, I realized that the GUI code never uses anything from Boost and hence doesn't have include directory configured for Boost. `cryptonote_basic/subaddress_index.h` includes some boost headers, and thus it'd force me to properly configure the Boost include path in the Qt project file, which seems like a PITA since there's no `find_package` for Qt. Therefore, a simple solution would be to just use the pair of `uint32_t` instead of `subaddress_index`.